test.tf

```diff
@@ -108,24 +123,99 @@ resource "azurerm_postgresql_server" "example" {
   sku_name                     = "B_Gen5_2"
   storage_mb                   = 5120
   backup_retention_days        = 7
-  geo_redundant_backup_enabled = false
+  geo_redundant_backup_enabled = true
   auto_grow_enabled            = true
   administrator_login          = "DKFJnisdfu"
   administrator_login_password = "Aa12345678"
   version                      = "9.5"
   ssl_enforcement_enabled      = true
+
+  identity {
+    type = "SystemAssigned"
+  }
 }
 
 resource "azurerm_postgresql_configuration" "thrtottling_config" {
   name                = "connection_throttling"
   resource_group_name = "testrg"
   server_name         = azurerm_postgresql_server.example.name
-  value               = "off"
+  value               = "on"
 }
 
 resource "azurerm_postgresql_configuration" "example" {
   name                = "log_checkpoints"
   resource_group_name = "testrg"
   server_name         = azurerm_postgresql_server.example.name
-  value               = "off"
+  value               = "on"
+}
+data "azurerm_client_config" "example_clientconfig" {}
+resource "azurerm_postgresql_active_directory_administrator" "example_adconfig" {
+  server_name         = azurerm_postgresql_server.example.name
+  resource_group_name = azurerm_resource_group.resource_group.name
+  login               = "sqladmin"
+  tenant_id           = data.azurerm_client_config.example_clientconfig.tenant_id
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure that public network access is disabled for Azure MySQL Server**
 
**Vulnerability**
Azure MySQL Server has public network access enabled, this may lead to unauthorized access/data leak.
 
**Remediation**
Ensure that your Microsoft Azure MySQL Servers are accessible only through private endpoints instead of public IP addresses or service endpoints, in order to eliminate the exposure from the public Internet.

source: accurics-beta